### ISSUE TITLE
False-positive with `RegExp.escape`

### ISSUE THREAD

**Issue by silverwind** [dustinspecker/eslint-plugin-no-use-extend-native#157]
The rule currently incorrectly flags the [`RegExp.escape`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/escape) static method:

```ts
RegExp.escape("string")
```
